pyhoma/enums/__init__.py

```diff
@@ -0,0 +1,8 @@
+# flake8: noqa: F401, F403
+
+from .command import *
+from .execution import *
+from .gateway import *
+from .general import *
+from .state import *
+from .ui import *
```

Comment:
So it was useful finally?

Reply:
The only reason why I did this is to have everything as an import from `enums` (which is also non breaking :D).